docs/user/gcp/install_upi.md

```diff
@@ -277,75 +296,66 @@ resources:
 - name: cluster-firewall
   type: 03_firewall.py
   properties:
+    allowed_external_cidr: '0.0.0.0/0'
     infra_id: '${INFRA_ID}'
     cluster_network: '${CLUSTER_NETWORK}'
     network_cidr: '${NETWORK_CIDR}'
-    master_nat_ip: '${MASTER_NAT_IP}'
-    worker_nat_ip: '${WORKER_NAT_IP}'
 - name: cluster-iam
   type: 03_iam.py
   properties:
     infra_id: '${INFRA_ID}'
 EOF
 ```
+- `allowed_external_cidr`: limits access to the cluster API and ssh to the bootstrap host. (for example External: 0.0.0.0/0, Internal: 10.0.0.0/16)
```

Comment:
`Internal: 10.0.0.0/16` > `Internal: $NETWORK_CIDR`

Reply:
Great idea. I'll roll that into #3278 so we can get this merged.